raptorWeb/raptorbot/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```

Comment:
Currently working on fixing up the actual bot itself, but we can create views here for an Admin interface of sorts for the bot. or we could just use the Django admin, unsure. In the grand scheme of things, I think a larger admin interface of sorts would be better than using Django Admin

Reply:
This will be addressed later on down the line, when a larger admin interface for the whole app is constructed